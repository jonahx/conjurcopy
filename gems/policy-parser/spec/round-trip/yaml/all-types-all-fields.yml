- !role
  kind: jenkins-folder
  id: team-1

- !resource
  kind: variable
  id: my-secret

- !user
  id: alice
  annotations:
    skills: [ red-tape-cutting, long-jacket-wearing ]

- !user bob

- !group
  id: developers
  
- !group operations

- !host myapp-01.example.com

- !host
  id: myapp-02.example.com

- !layer myapp

- !layer
  id: yourapp

- !variable db-password

- !variable
  id: bank-vault-combination
  kind: combination
  mime_type: text/plain

- !webservice release-bot

- !webservice
  id: quake2-server

  