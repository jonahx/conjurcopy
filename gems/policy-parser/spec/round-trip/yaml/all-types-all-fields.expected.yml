---
- !role
  id: team-1
  kind: jenkins-folder
- !resource
  id: my-secret
  kind: variable
- !user
  annotations:
    skills:
    - red-tape-cutting
    - long-jacket-wearing
  id: alice
- !user
  id: bob
- !group
  id: developers
- !group
  id: operations
- !host
  id: myapp-01.example.com
- !host
  id: myapp-02.example.com
- !layer
  id: myapp
- !layer
  id: yourapp
- !variable
  id: db-password
- !variable
  id: bank-vault-combination
  kind: combination
  mime_type: text/plain
- !webservice
  id: release-bot
- !webservice
  id: quake2-server
